src/args.jl

```diff
@@ -434,6 +434,7 @@ const _plot_defaults = KW(
     :thickness_scaling   => 1,
     :display_type        => :auto,
     :warn_on_unsupported => true,
+    :plotarea => :auto,
```

Comment:
We already have a notion for plotarea with a slightly different meaning
https://github.com/JuliaPlots/Plots.jl/blob/6266ff8d08e98f2f3980731174cb1e2a631fd49a/src/types.jl#L39

IIIUC, your plotarea is the whole area covered by one subplot, whereas the subplot field is the area of the axes and data without labels and margins and stuff

Reply:
Ah hmm. What would you recommend?